examples/advanced_diffusion_training/train_dreambooth_lora_sdxl_advanced.py

```diff
@@ -1335,6 +1337,10 @@ def save_model_hook(models, weights, output_dir):
                 text_encoder_lora_layers=text_encoder_one_lora_layers_to_save,
                 text_encoder_2_lora_layers=text_encoder_two_lora_layers_to_save,
             )
+        if args.train_text_encoder_ti:
+            embedding_handler.save_embeddings(
+                f"{output_dir}/{args.output_dir}_emb.safetensors",
+            )
```

Comment:
```suggestion
            embedding_handler.save_embeddings(f"{output_dir}/{args.output_dir}_emb.safetensors")
```

I have a feeling we might be able to fit this in a single line.